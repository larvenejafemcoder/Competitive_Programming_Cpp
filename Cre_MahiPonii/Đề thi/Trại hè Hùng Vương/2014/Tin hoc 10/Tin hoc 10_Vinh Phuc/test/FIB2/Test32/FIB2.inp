100
31 1217592
15 132
14 463
9 14
3 1
15 926
39 13422555
14 320
14 419
29 9470
1 1
21 4577
20 8723
23 26099
31 1090026
35 5330354
35 13630354
21 4175
32 3430146
32 1974376
38 58180690
4 3
33 4264898
40 155110418
11 139
27 30966
29 826345
18 1748
25 41891
45 890129453
27 21260
41 240182326
17 891
3 3
21 6968
5 2
39 46446443
42 26739695
15 644
11 74
20 7979
28 212521
28 440098
33 3813925
41 89739276
16 922
22 27738
18 2188
4 5
32 1481512
39 80380796
15 261
7 18
14 422
35 2951938
45 1087186559
10 50
4 1
22 19608
17 1890
40 93271908
35 3922737
33 2125744
26 29692
36 7868956
34 4051525
1 1
8 6
23 21931
8 20
6 1
34 5523291
16 401
25 87399
8 14
33 564064
22 23957
5 8
4 4
35 7051140
6 2
27 181252
21 10258
32 2275773
7 19
24 5485
20 791
37 19940769
31 1134396
9 26
14 482
40 144254516
22 22557
25 117220
3 2
30 672575
36 9806869
18 3075
37 32880130
16 947
