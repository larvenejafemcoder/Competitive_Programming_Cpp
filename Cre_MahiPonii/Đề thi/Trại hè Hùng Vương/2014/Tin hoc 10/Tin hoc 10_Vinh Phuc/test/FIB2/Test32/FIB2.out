465079
51
177
5
0
354
5126960
122
160
3617
0
1748
3332
9969
416353
2036014
5206332
1595
1310199
754145
22223046
1
1629046
59246908
53
11828
315636
668
16001
339999197
8121
91741485
340
1
2662
1
17740963
10213655
246
28
3048
81176
168103
1456790
34277353
352
10595
836
2
565887
30702732
100
7
161
1127540
415268314
19
1
7490
722
35626699
1498352
811962
11341
3005674
1547545
0
2
8377
8
1
2109710
153
33384
5
215453
9151
3
2
2693296
1
69232
3918
869268
7
2095
302
7616696
433301
10
184
55100322
8616
44774
1
256901
3745891
1175
12559092
362
